src/api/app/models/workflow.rb

```diff
@@ -84,7 +85,7 @@ def supported_filters
 
   def destroy_target_projects
     # Do not process steps for which there's nothing to do
-    processable_steps = steps.reject { |step| step.instance_of?(::Workflow::Step::ConfigureRepositories) }
+    processable_steps = steps.reject { |step| step.instance_of?(::Workflow::Step::ConfigureRepositories) || step.instance_of?(::Workflow::Step::RebuildPackage) }
```

Comment:
We should improve it later. Either having a field `skip_destroy_restore` or via class heritage... could it become a card in the backlog @dmarcoux ?